spk/rutorrent/src/installer.sh

```diff
@@ -130,14 +164,16 @@ postinst ()
         fi
 
         if [ "${wizard_disable_openbasedir}" == "true" ] && [ "${APACHE_USER}" == "http" ]; then
-            sed -i -e "s|^open_basedir.*|open_basedir = none|g" /etc/php/conf.d/user-settings.ini
-            initctl restart php-fpm > /dev/null 2>&1
+            if [ -f "/etc/php/conf.d/user-settings.ini" ]; then
```

Comment:
Not sure about what this was for (given previous logics) but for DSM 6.0 it would cause error messages to be issued since file wouldn't exist.